### ISSUE TITLE
npm WARN deprecated slack-client@1.5.1: Use @slack/client instead, this package is no longer maintained

### ISSUE THREAD

**Issue by infernix** [camwhite/slack-xmpp#3]
as per subject.

**Comment by camwhite** [comment]
Yes it's true, @infernix.. I have a lot on my plate feel free to open a PR for the upgrade! As it stands it is working for me, so not a priority rn unfortunately :shipit:

**Comment by infernix** [comment]
I couldn't get this to work well enough for my use case so unfortunately no PR for me. Sorry! :)
